### PR TITLE
[jsk_robot_startup] add google_chat function in smach_to_mail.py

### PR DESCRIPTION
**Pull request by mqcmd196**
related to https://github.com/jsk-ros-pkg/jsk_robot/pull/1558
I added `google_chat_ros` actionlib client in `smach_to_mail.py` .

### CODE REVIEW

**Comment by k-okada** [comment]
@mqcmd196 thank you.
- please update system configuration figure (see  https://github.com/jsk-ros-pkg/jsk_robot/pull/1558#issuecomment-1209050229  for example)
- please plaste  google chat output image
(see https://twitter.com/fetch15jsk for example with https://drive.google.com/file/d/14jqlB1QEJZGbrcRHRcQ3g73rbHMJ8hhf/view?usp=sharing , the image in this bag file is not good, so you can craete good exampe bag file by your self)

**Comment by k-okada** [comment]
@mqcmd196  > please paste google chat output image

**Comment by tkmtnt7000** [comment]
@mqcmd196 
Readme of `smach_to_mail.py` has already been merged, but if there is something wrong with description about google_chat_ros, could you please fix it?
https://github.com/k-okada/jsk_robot/tree/smach_to_state/jsk_robot_common/jsk_robot_startup#scriptssmach_to_mailpy

**Comment by mqcmd196** [comment]
Sorry for being late, I confirmed it works like this
chat_notification

**Comment by mqcmd196** [comment]
@tkmtnt7000 I see, I'll update the doc.

**Comment by mqcmd196** [comment]
I updated the README in https://github.com/k-okada/jsk_robot/pull/72/commits/9d381967f536264fdab0e584108fb3770118507c

**Comment by mqcmd196** [comment]
sorry I missed merging upstream, please wait for fixing...

**Comment by mqcmd196** [comment]
updated, it's ready for merged

File: jsk_robot_common/jsk_robot_startup/package.xml [hunk 1]
```diff
@@ -22,8 +22,10 @@
   <run_depend>actionlib_msgs</run_depend>
   <run_depend>app_manager</run_depend>
   <run_depend>dynamic_reconfigure</run_depend>
+  <run_depend>gdrive_ros</run_depend>
```
Comment (k-okada) [inline jsk_robot_common/jsk_robot_startup/package.xml:25]: do we need this?

**Review by k-okada — COMMENTED** [review]
@mqcmd196 thank you 
pllease add `jsk_robot_startup/launch/smach_to_mail.launch` and accespt `use_mail` `use_twitter`, `use_google_chat` argumetn, also update doc/

File: jsk_robot_common/jsk_robot_startup/package.xml (hunk 1 above)
Comment (mqcmd196) [reply jsk_robot_common/jsk_robot_startup/package.xml:25]: Sorry, it is already written in `google_chat_ros`

**Comment by mqcmd196** [comment]
I rebased `origin/master`, so close this PR.

**Pull request closed without merging**